Assets/Scripts/Inputs/ControlPoint.cs

```diff
@@ -32,11 +32,12 @@ void Update()
 
     public void ReceiveTouchInput(Vector2 moveInput)
     {
+        var rotationSpeed = _rotationSpeed / 100000;
         var smallestScreenDim = Screen.currentResolution.height < Screen.currentResolution.width
             ? Screen.currentResolution.height
             : Screen.currentResolution.width;
```

Comment:
this part could probably be moved to void Start() (one of the functions that are created by default, when you create a script). That way you'll only calculate it once and not for every input, which is unnecessary. But also if we're allowing players to rotate screen mid play, it will introduce a bug where the input sensitivity wouldn't change? (not sure about that)

Reply:
good idea, tried, everything worked with no bugs